app/services/record_declaration.rb

```diff
@@ -156,10 +156,23 @@ def declaration_parameters
       declaration_type:,
       cpd_lead_provider:,
       delivery_partner:,
+      mentor_user_id: mentor&.id,
       user: participant_identity.user,
     }
   end
 
+  def mentor
+    return nil unless participant_profile.ect?
+
+    induction_record = Induction::FindBy.call(
```

Comment:
having a deeper think about this, we probably need the time range between the start date AND end date of an induction record, if the end date is empty it means that one is the latest (in some cases 😅). Let's chat tomorrow